scripts/enum/mob_mod.lua

```diff
@@ -78,7 +78,7 @@ xi.mobMod =
     ALLI_HATE              = 68, -- Range around target to add alliance member to enmity list.
     NO_LINK                = 69, -- If set, mob cannot link until unset.
     NO_REST                = 70, -- Mob cannot regain hp (e.g. re-burrowing antlions during ENM).
-    LEADER                 = 71, -- Used for mobs that follow a defined "leader", such as Ul'xzomit mobs.
+    LEADER                 = 71, -- Used for mobs that follow a defined "leader". Positive designates a leader and number of follows. Negative is the leader ID relative to the mob.
```

Comment:
Is it ALWAYS the case that the IDs of leaders will be lower than the followers, am I understanding that right?

Reply:
Yes, the ID of leaders is always lower than the followers. I reworded this comment to hopefully clarify things a bit better.

`Used for mob following. Positive number is how many followers a leader has. A negative number is the distance from this mob ID to the leader ID.`